build.gradle

```diff
@@ -276,6 +276,12 @@ subprojects {
         }
     }
 
+    // Create an empty classes directory to satisfy Sonar
+    // This is necessary since Sonar requires class directories to exist if the Java plugin is applied, which
+    // is currently done on all sub-projects. Sub-projects without any classes, like some of those under
+    // :repose-aggregator:artifacts, will break Sonar without this line.
+    file("$buildDir/classes/main").mkdirs()
```

Comment:
Add a line about looking if ithis can be removed in future versions of the plugin